crates/aptos-crypto/src/bls12381/bls12381_keys.rs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module provides APIs for private keys and public keys used in BLS multi-signatures
+//! implemented on top of BLS12-381 elliptic curves (https://github.com/supranational/blst).
+//!
+//! The `PublicKey` struct is used to represent both the public key of an individual signer
+//! as well as the aggregate public key of several signers. Before passing such a struct as an
+//! argument the caller should always ensure it has verified its proof-of-possession via
```

Comment:
1. such a struct → this struct
2. what is "it" referring to? caller? If yes, I suggest "the caller should always ensure that the caller has verified the proof-of-possession for all the public keys being represented by this struct."